src/models/users.js

```diff
@@ -27,7 +26,15 @@ class User {
     }
 
     static createUser(newUser, result) {
-        db.query(`INSERT INTO users VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?)`, ['', newUser.email, newUser.first_name, newUser.last_name, newUser.hashPassword, newUser.phone, newUser.address, newUser.is_admin, ''], (err, res) => {
+        var date_ob = new Date();
```

Comment:
If this code is trying to set date for the created_at and updated_at timestamps, then it's not needed as the database can take care of that for us automatically.

Reply:
Yeah the database will automatically add the time if the user data is created or updated so we don't need to do this

Reply:
Mine wasn't adding up.